core/src/main/scala/wartremover/contrib/warts/SymbolicName.scala

```diff
@@ -0,0 +1,27 @@
+package org.wartremover
+package contrib.warts
+
+object SymbolicName extends WartTraverser {
+
+  private val validChars = "[a-zA-Z_]+".r
+
+  def apply(u: WartUniverse): u.Traverser = {
+    import u.universe._
+
+    def isSymbolic(name: String) = validChars.replaceAllIn(name, "").length > 2
```

Comment:
`===` in `Equals` description is a suggestion. And I don't think it's a good name either. Since the implementation isn't symmetric, I would use `is`, for example.